python/pyspark/sql/streaming/readwriter.py

```diff
@@ -1344,20 +1344,29 @@ def foreachBatch(self, func: Callable[["DataFrame", int], None]) -> "DataStreamW
 
         .. versionadded:: 2.4.0
 
+        .. versionchanged:: 3.5.0
+        Supports Spark Connect.
+
```

Comment:
Found we missed a lot of these, added a separate PR for all of them:
https://github.com/apache/spark/pull/42097

Reply:
Thanks!